src/components/ScreenContainer.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+
+const ScreenContainer = ({ children }) => {
```

Comment:
The same about indentation and ESLint, comments about that that I left in the MainStackNavigator.js file and about the style files.